server/controllers/mainController.js

```diff
@@ -41,3 +42,26 @@ exports.novelsPage = async (req, res) => {
 		err => console.log(err)
 	}
 }
+
+exports.shortsPage = async (req, res) => {
+	try {
+		const currentShortStorySlug = req.params.storyTitle
+		const text = await ShortStory.findOne({
```

Comment:
Does this get the story object from the db? :) If so, story could be a clearer variable name, since there's storyText further down.

Reply:
Thank you, noted and updated.